src/internal/connector/exchange/restore_test.go

```diff
@@ -125,6 +125,41 @@ func (suite *ExchangeRestoreSuite) TestRestoreEvent() {
 		fault.New(true))
 	assert.NoError(t, err, clues.ToCore(err))
 	assert.NotNil(t, info, "event item info")
+
```

Comment:
This test is a bit out of date.  While you're extending it, would you mind refactoring its implementation to a [table driven test](https://dave.cheney.net/2019/05/07/prefer-table-driven-tests)?  You should be able to copy boilerplate from much of the surrounding tests, if needed.

Reply:
Sounds good!